src/main/java/uk/gov/justice/digital/hmpps/keyworker/integration/casenotes/CaseNotesApiClient.kt

```diff
@@ -26,6 +28,24 @@ class CaseNotesApiClient(
       }.retryRequestOnTransientException()
       .block()!!
 
+  fun getUsageByPersonIdentifiers(
+    requests: List<UsageByPersonIdentifierRequest>,
+  ): List<Pair<UsageByPersonIdentifierRequest, NoteUsageResponse<UsageByPersonIdentifierResponse>>> =
+    Flux
+      .fromIterable(requests)
+      .flatMap({
+        webClient
+          .post()
+          .uri("/case-notes/usage")
+          .bodyValue(it)
+          .retrieve()
+          .bodyToFlux<NoteUsageResponse<UsageByPersonIdentifierResponse>>()
+          .retryRequestOnTransientException()
```

Comment:
ManageUsersClient.getUsersDetails has a similar pattern but doesn't look like it has retry on it. Could you check if they can both use the same pattern